### PR TITLE
update parse_cli documentation with more examples

### PR DESCRIPTION
**Pull request by privateip**

### CODE REVIEW

File: docs/docsite/rst/playbooks_filters.rst [hunk 1]
```diff
@@ -356,6 +357,53 @@ will parse the output from the ``show vlan`` command.::
 The spec file above will return a JSON data structure that is a list of hashs
```
Comment (gundalow) [inline docs/docsite/rst/playbooks_filters.rst]: hashs -> hashes

**Review by dharmabumstead — APPROVED** [review]

File: docs/docsite/rst/playbooks_filters.rst (hunk 1 above)
Comment (privateip) [reply docs/docsite/rst/playbooks_filters.rst]: fixed

**Comment by abadger** [comment]
merged to devel.  @gundalow if you think this should be backported, feel free to cherrypick.

**Comment by gundalow** [comment]
This is new in 2.4, so no need to backport

**Pull request merged**